.circleci/config.yml

```diff
@@ -39,7 +39,12 @@ jobs:
           paths:
             - ./venv
 
-      # run tests
+      # run autostyle!
+      - run:
+          name: run autostyle
+          command: poetry run pre-commit run --all-files --show-diff-on-failure
```

Comment:
The only thing really to note here is that pre-commit (like other dev tools) should really be managed by poetry too - that will keep everything a lot saner.

If you have a globally-installed pre-commit that's fine but IMO it just makes everything way harder to debug when there's a subtle version mismatch.